src/main/java/io/dropwizard/kafka/KafkaProducerFactory.java

```diff
@@ -74,6 +74,9 @@
     @JsonProperty
     protected int batchSize = 16384;
 
+    @JsonProperty
+    protected Optional<Boolean> enableIdempotence = Optional.empty();
```

Comment:
Instead of using an `Optional` here, can we use whatever the default Boolean value is (whether the default is true/false?)

Reply:
Thanks for taking a look so quickly! Also - would you mind if I went ahead and added a few more configs?